libgit/repo.go

```diff
@@ -21,11 +23,12 @@ import (
 )
 
 const (
-	kbfsRepoDir         = ".kbfs_git"
-	kbfsConfigName      = "kbfs_config"
-	kbfsConfigNameTemp  = "._kbfs_config"
-	gitSuffixToIgnore   = ".git"
-	kbfsDeletedReposDir = ".kbfs_deleted_repos"
+	kbfsRepoDir              = ".kbfs_git"
+	kbfsConfigName           = "kbfs_config"
+	kbfsConfigNameTemp       = "._kbfs_config"
+	gitSuffixToIgnore        = ".git"
+	kbfsDeletedReposDir      = ".kbfs_deleted_repos"
```

Comment:
I know this was added in a previous PR, but I was just thinking what would  happen if `git push` takes longer than an hour (i.e. `minDeletedAgeForCleaning`), and repo delete happens in between. I suppose git-remote-keybase will find out about conflict and rename the repo folder, and only the original that was moved into `.kbfs_deleted_repos` would be deleted, meaning stuff from the eventually finished super slow `git push` would still stay inside `.kbfs_git` as a renamed folder with a conflict suffix so if user needs they can still find them back. Does it sound correct?

Reply:
If a repo is deleted (i.e., renamed to the `.kbfs_deleted_repos` directory), CR will apply the push changes to that renamed directory.  No conflict names will be used, and the newly pushed data will get cleaned as normal with the rest of the repo.

If the repo is deleted AND then cleaned up all during the push (that's quite a push!), then CR will recreate the pushed files in their renamed location under `.kbfs_deleted_repos`, also without a conflict name (since there's nothing left to conflict with).  Though we don't have a test for this, so I'm not 100% sure -- it could get recreated under the old `.kbfs_git` repo name.  I can add a test later, but either way it seems super unlikely to be an issue.  If it becomes an issue, we can add a check after CR completes to make sure the repo's config file still exists, and if not, we fail the push so the user sees an error.

Reply:
Thanks for the explanation. I was mostly worried about the situation where user sees the push succeeds, but it ends up in `.kbfs_deleted_repos` and later on gets cleaned up because another team member deleted the repo. But I guess in that case since delete happened, it should be expected that it's gone anyway.